pkgs/applications/graphics/ImageMagick/default.nix

```diff
@@ -59,6 +60,13 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-NrTIx1OvwPIeVlA39hGkXZ2Atk4FCsU3/55SZeSc40E=";
   };
 
+  patches = [
+    (fetchpatch2 {
+      url = "https://github.com/ImageMagick/ImageMagick/commit/bf5650f0dd41b500102a129d6867cb568f4edee4.patch";
```

Comment:
doesn't `fetchpatch2` need to use append `?full_index=1` to the GitHub url to get a reproducible patch?